src/main/java/org/broadinstitute/hellbender/utils/Utils.java

```diff
@@ -515,11 +517,64 @@ public static boolean endsWith(final byte[] big, final byte[] suffix) {
      * @return the same object
      * @throws IllegalArgumentException if a {@code o == null}
      */
-    public static <T> T nonNull(T object) throws IllegalArgumentException {
+    public static <T> T nonNull(final T object) throws IllegalArgumentException {
```

Comment:
This should call into `nonNull(T, String)` below with the default message `"Null object is not allowed here."`